utils/calico-uts.Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:14.04
+
+MAINTAINER Alex Chan <alexchan@projectcalico.org>
+
+RUN apt-get update
+RUN echo "APT::GET::Assume-Yes \"true\";" >> /etc/apt/apt.conf
+
+# tox installs python-posix-spawn and python-etcd from Git
+RUN apt-get install git
```

Comment:
Would it be okay to pull these commands into a shell script, copy that shell script into the image, and just run that script? I'm not keen on stuffing them into one line -- it makes it harder to see what's actually needed.